.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,15 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
+about: Create a report to help us improve Invidious
+title: '[Bug] '
 labels: bug
 assignees: ''
 
 ---
 
 <!-- Please use the search function to check if the bug you found has already been reported by someone else -->
 <!-- If you want to suggest a new feature please use "Feature request" instead -->
+<!-- If you want to suggest an enhancement to a new feature please use "Enhancement" instead -->
```

Comment:
>If you want to suggest an enhancement to a new feature please use "Enhancement" instead

Should be: 

If you want to suggest an enhancement to an **existing** feature please use "Enhancement" instead